go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go

```diff
@@ -546,7 +546,55 @@ func TestBuffering(t *testing.T) {
 			`type:DDL ddl:"alter table packet_test change val val varchar(128)" `,
 		}},
 	}}
-	runCases(t, nil, testcases)
+	runCases(t, nil, testcases, "")
+}
+
+func TestBestEffortNameInFieldEvent(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	filter := &binlogdatapb.Filter{
+		BestEffortNameInFieldEvent: true,
+		Rules: []*binlogdatapb.Rule{{
+			Match: "/.*/",
+		}},
+	}
+	// Modeled after vttablet endtoend compatibility tests.
+	execStatements(t, []string{
+		"create table vitess_null(id int, val varbinary(128), primary key(id))",
```

Comment:
why vitess_null? that seems like a weird table name when the test has nothing to do with NULL values.

Reply:
i picked a table from existing test TestTypes

Reply:
Then I suggest renaming it to vitess_test or field_event_test

Reply:
done!